pkg/network/dns/ebpf.go

```diff
@@ -82,4 +83,6 @@ func (e *ebpfProgram) Init() error {
 		ConstantEditors:           constantEditors,
 		DefaultKprobeAttachMethod: kprobeAttachMethod,
 	})
+	ebpfcheck.AddNameMappings(e.Manager)
```

Comment:
do we still want to do this if `err != nil`?

Reply:
Ah, nope. Fixing.

Reply:
What about other places?

Reply:
All the other places were already correct. Do you see another one?

Reply:
I don't see any others, at least for the network code.